### PR TITLE
add pyMLST

### PR DESCRIPTION
**Pull request by Kincekara**
pyMLST is a tool that can make cg/wgMLST analysis as well as MLST typing. It may be a good addition to staphb

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: pymlst/2.1.5/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    python3-biopython &&\
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# get blat from builder
+COPY --from=blat /usr/local/bin/* /usr/local/bin/
+
+# install pymlst
+RUN pip3 install --no-cache pymlst==${PYMLST_VER}
+
+CMD ["pyMLST", "-h"]
```
Comment (erinyoung) [inline pymlst/2.1.5/Dockerfile:50]: Is 'pyMLST' actually executable? All the examples I'm seeing use `claMLST` or `wgMLST`.

Comment (Kincekara) [reply pymlst/2.1.5/Dockerfile:50]: You are right! It seems pyMLST is for configuration and log level which is not so meaningful alone.

**Comment by Kincekara** [comment]
@erinyoung I have modified the CMD part. Is there anything else you want to change?

**Comment by erinyoung** [comment]
My apologies for being confused! I think it looks good.

**Review by erinyoung — APPROVED** [review]

**Comment by erinyoung** [comment]
Thank you for putting this together. You can check the status of the deploy here : https://github.com/StaPH-B/docker-builds/actions/runs/8236051446

**Pull request merged**